src/docs/index.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Known pages in the app. Highest order context.
+ */
+export enum DocsPageName {
+  HOME_PAGE = 'home-page',
+}
+
+/**
+ * Sections. Disambiguate low-level elements that may share a name.
+ * eg a `back` button in a modal will have the same `element`,
+ * but a different `section`.
+ */
+export enum DocsSectionName {
+  BOTTOM_MENU_LINKS = 'Bottom Menu Links',
+  DEVELOPER_LINKS = 'Developer Links',
+  GETTING_STARTED_LINKS = 'Getting Started Links',
+  QUICK_LINKS = 'Quick Links',
+  WELCOME_LINKS = 'Welcome Links',
+}
+
+/**
+ * Known element names for analytics purposes.
+ * Use to identify low-level components given a TraceContext
+ */
+export enum DocsElementName {
```

Comment:
Are all of these home page elements now? Should we rename to help separate from other potential element groups? e.g. `DocsHomepageElementName`